test_scripts/Defects/6_1/1918_Multiple_apps_activation_during_active_embedded_audio_source.lua

```diff
@@ -0,0 +1,145 @@
+local common = require("user_modules/sequences/actions")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Applications Configuration ]]
+config.application1.registerAppInterfaceParams.appHMIType = { "MEDIA" }
+config.application1.registerAppInterfaceParams.isMediaApplication = true
+config.application2.registerAppInterfaceParams.appHMIType = { "NAVIGATION" }
+config.application2.registerAppInterfaceParams.isMediaApplication = false
+config.application3.registerAppInterfaceParams.appHMIType = { "DEFAULT" }
+config.application3.registerAppInterfaceParams.isMediaApplication = false
+
+--[[ Local Functions ]]
+local function activateApp(pAppId)
+  if not pAppId then pAppId = 1 end
+  local requestId = common.getHMIConnection():SendRequest("SDL.ActivateApp", { appID = common.getHMIAppId(pAppId) })
+  common.getHMIConnection():ExpectResponse(requestId)
+  common.getMobileSession(pAppId):ExpectNotification("OnHMIStatus")
+  :ValidIf(function(_, data)
+    if config["application"..pAppId].registerAppInterfaceParams.isMediaApplication == false and
+      config["application"..pAppId].registerAppInterfaceParams.appHMIType[1] == "DEFAULT"
+      then
+        return data.payload.audioStreamingState == "NOT_AUDIBLE" and
+          data.payload.hmiLevel == "FULL" and
+          data.payload.systemContext == "MAIN"
+      else
+        return data.payload.audioStreamingState == "AUDIBLE" and
+          data.payload.hmiLevel == "FULL" and
+          data.payload.systemContext == "MAIN"
+    end
+  end)
+end
+
+local function deactivateApp(pAppId)
+  if not pAppId then pAppId = 1 end
+  common.getHMIConnection():SendNotification("BasicCommunication.OnAppDeactivated", { appID = common.getHMIAppId(pAppId) })
+  common.getMobileSession(pAppId):ExpectNotification("OnHMIStatus")
+  :ValidIf(function(_, data)
+    if config["application"..pAppId].registerAppInterfaceParams.isMediaApplication == true
+      then
+        return data.payload.audioStreamingState == "AUDIBLE" and
+          data.payload.videoStreamingState == "NOT_STREAMABLE" and
+          data.payload.hmiLevel == "LIMITED" and
+          data.payload.systemContext == "MAIN"
+      else
+        return data.payload.audioStreamingState == "NOT_AUDIBLE" and
+          data.payload.videoStreamingState == "NOT_STREAMABLE" and
+          data.payload.hmiLevel == "BACKGROUND" and
+          data.payload.systemContext == "MAIN"
+    end
+  end)
+end
+
+local function embededAudioActivated()
+  common.getHMIConnection():SendNotification("BasicCommunication.OnEventChanged",
+    { eventName = "AUDIO_SOURCE", isActive = true })
+  common.getMobileSession(1):ExpectNotification("OnHMIStatus", {
+    hmiLevel = "BACKGROUND", audioStreamingState = "NOT_AUDIBLE", videoStreamingState = "NOT_STREAMABLE"
+  })
+  common.getMobileSession(2):ExpectNotification("OnHMIStatus", {
+    hmiLevel = "LIMITED", audioStreamingState = "AUDIBLE", videoStreamingState = "NOT_STREAMABLE"
+  })
```

Comment:
```suggestion
  -- hmiLevel = "LIMITED", audioStreamingState = "AUDIBLE", videoStreamingState = "STREAMABLE"
  common.getMobileSession(2):ExpectNotification("OnHMIStatus"):Times(0)
```
App would be in `LIMITED`, `AUDIBLE`, `STREAMABLE`